### PR TITLE
Update a10-thunder.yml

### PR DESCRIPTION
**Pull request by al3kh**
Added HA OIDs
@thezackm can you please approve

### CODE REVIEW

**Comment by al3kh** [comment]
@thezackm hi, did you get a chance to take a look

**Comment by al3kh** [comment]
@Mesverrum hi, can you please approve this. thanks

File: profiles/kentik_snmp/a10_networks/a10-thunder.yml [hunk 1]
```diff
@@ -479,6 +479,32 @@ metrics:
...
           name: axVirtualServerPortStatName
           OID: 1.3.6.1.4.1.22610.2.4.3.4.4.1.1.4
+  # A table containing HA information.
+  - MIB: A10-AX-MIB
+    table:
+      name: axHAGroupStatusTable
+      OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2
+    symbols:
+      - name: axHAGroupStatusEntry
+        OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2.1
```
Comment (thezackm) [inline profiles/kentik_snmp/a10_networks/a10-thunder.yml:489]: the `*Entry` field in a MIB table is a placeholder for our needs and should not be part of this profile; please remove.

File: profiles/kentik_snmp/a10_networks/a10-thunder.yml [hunk 2]
```diff
@@ -479,6 +479,32 @@ metrics:
...
+      - name: axHAGroupID
+        OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2.1.1
```
Comment (thezackm) [inline profiles/kentik_snmp/a10_networks/a10-thunder.yml:491]: this is a non-volatile ID field (similar to a name) and should be used as a `metric_tag` value for this table

**Review by thezackm — CHANGES_REQUESTED** [review]
@al3kh - please see changes requested

**Comment by al3kh** [comment]
@thezackm Hi, made the requested changes.

File: profiles/kentik_snmp/a10_networks/a10-thunder.yml [hunk 3]
```diff
@@ -493,3 +515,6 @@ metric_tags:
   - column:
       name: axSysAFleXEngineVersion
       OID: 1.3.6.1.4.1.22610.2.4.1.6.4.0
+  - column:
+      name: axHAGroupID
+      OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2.1.1
```
Comment (thezackm) [inline profiles/kentik_snmp/a10_networks/a10-thunder.yml:520]: @al3kh - this needs to be targeted to the table; this `metric_tags` section is the global one that applies to the entire profile. There are examples of the correct indentation level and syntax throughout this profile; the first is on lines 179-183.

at a high level:

```yaml
metrics:
  - MIB: $MIB_NAME
    table: ... # table overview
    symbols: ... # metrics to collect from this table
    metric_tags: ... # attributes to decorate on top of the symbols defined in this table
...

# global metric_tags that apply to all scalar OIDs in this profile
metric_tags: ...
```

**Review by thezackm — CHANGES_REQUESTED** [review]
@al3kh - one more small edit to move the `metric_tag` into the proper section

**Review by thezackm — APPROVED** [review]

**Pull request merged**